oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+- `Code Committer Rights & Responsibilities <https://openedx.atlassian.net/wiki/spaces/COMM/pages/1529675973/Rights+Responsibilities+for+Code+Contributors>`_
+
+Note that roles vary in what they need from members; the wiki will be the most up to date source
+of what each role entails and what types of commitments are expected.
+
+.. _program administration:
+
+Core Contributor Program Administration
+---------------------------------------
+
+The Core Contributor Program is administered by The Center for Re-imagining
+Learning (tCRIL). One or more persons at tCRIL (the "Program administrator(s)")
+work with the community to make sure the program runs smoothly: this entails
+facilitating communication amongst community members, `selecting new members`_,
+and ensuring that existing members renew their commitment to the community and
+abide by the Code of Conduct. Program administrators are empowered to monitor
+Core Contributor conduct, particularly whether they follow the "3 Cs"
+(Commitment, Conduct, Caliber), and may issue reminders, suspensions, and even
+removal from the Program. Program administrators can be contacted at
+``cc-program-admins@tcril.org``.
+
+The tCRIL administrator(s) are the point people for making sure that needed
+agreements are signed - this includes a Program Agreement between the CC and
+tCRIL (sent to you by a Program Administrator); for code contributors, an
+additional CLA (signed when you make your first PR); and for new entities
+joining the community, an Entity Contributor Agreement (please communicate with
+a Program Administrator about this).
+
+We are additionally asking that all employers of CCs sign the `Revised
+Declaration of Commitment to the Core Contributor Program
+<https://openedx.atlassian.net/wiki/spaces/COMM/pages/3216900524>`_ to
+demonstrate that they are willing to engage with and commit resources to the
+Open edX project. We encourage a commitment of 20 hours/month from each of an
+organization's Core Contributors, however, we recognize that circumstances
+differ and a different commitment may be more appropriate. Before signing the
+agreement, please reach out to the Program Administrators to discuss.
+
+Questions about the Program can be directed to the tCRIL administrator(s) via
+the ``#core-contributors`` Slack room in the `Open edX Slack
+<https://openedx.slack.com/>`_.
+
+.. _selecting new members:
+
+Adding New Core Contributors
+----------------------------
+
+New Core Contributors are determined via a nomination process - CCs may nominate
+new members, or existing community members may put themselves forward for a role.
+
+Current CCs should always be on the lookout for contributors who demonstrate the
+"3 Cs" (Commitment, Conduct, and Caliber), follow the `Code of Conduct
+<https://open.edx.org/code-of-conduct/>`_, are respectful of other community members, are polite and
+welcoming to all in the community, and have been an overall active community member for some time.
```

Comment:
This sentence duplicates some of the first mention of the 3 Cs.

Reply:
I actually intentionally duplicated this, because I am not sure how many people simply read one section of an OEP and don't click through links. Also, there is a lot on the wiki that mentions the "3 C's" and I am frequently asked what those are. I'll make this a link if you still think it is better to do so.

Reply:
Duplication in code creates work to maintain, but duplication in documentation is not so bad :) I'm ok to keep this if you think it gets the point across better :)